uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/calendar-feeds.link-modal.module.js

```diff
@@ -1,19 +1,23 @@
 /* eslint-env amd */
+/* global Drupal */
 
-define([
-  'common/angular',
-  'leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.component',
-  'leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.service',
-  'leave-absences/calendar-feeds/link-modal/input-with-copy-button.directive',
-  'common/angularBootstrap',
-  'leave-absences/calendar-feeds/calendar-feeds.core'
-], function (angular, CalendarFeedsLinkModalComponent, CalendarFeedsLinkModalService,
-  inputWithCopyButton) {
-  return angular.module('calendar-feeds.link-modal', [
-    'ui.bootstrap',
-    'calendar-feeds.core'
-  ])
-    .component(CalendarFeedsLinkModalComponent.__name, CalendarFeedsLinkModalComponent)
-    .directive(inputWithCopyButton.__name, inputWithCopyButton)
-    .factory(CalendarFeedsLinkModalService.__name, CalendarFeedsLinkModalService);
-});
+(function (Drupal) {
+  define([
+    'common/angular',
+    'leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.component',
+    'leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.service',
+    'leave-absences/calendar-feeds/link-modal/input-with-copy-button.directive',
+    'common/angularBootstrap',
+    'leave-absences/calendar-feeds/calendar-feeds.core'
+  ], function (angular, CalendarFeedsLinkModalComponent, CalendarFeedsLinkModalService,
+    inputWithCopyButton) {
+    return angular.module('calendar-feeds.link-modal', [
+      'ui.bootstrap',
+      'calendar-feeds.core'
+    ])
+      .constant('SITE_HOST', Drupal.absoluteUrl('/'))
```

Comment:
Shouldn't this be defined in an even more generic module? Maybe shared settings?

Reply:
Done, defined in a separate module as per @AkA84 suggestion.